src/components/photopage/photopage.js

```diff
@@ -26,73 +26,73 @@ const officers = [
 		role: 'President',
 		name: 'Lea Blum',
 		id: 'lea',
-		description: 'Blurbs coming soon!'
+		description: `Hi there, I’m Lea and I’m one of the Hack Co-Presidents (along with Connie). Come hang out with me and the rest of hack at our events to hear my (sometimes) funny jokes and (always) hilarious puns.`
```

Comment:
Can you wrap the lines? The point of the template literal syntax (backticks) is so that you could break lines.

```suggestion
		description: `
			Hi there, I’m Lea and I’m one of the Hack Co-Presidents (along with
			Connie). Come hang out with me and the rest of hack at our events to hear
			my (sometimes) funny jokes and (always) hilarious puns.
		`
```

Revise similarly all other entries.